Add MASTG-TEST-0264, MASTG-TEST-0265, MASTG-DEMO-0038, MASTG-DEMO-0039 (#3246)
OWASP/owasp-mastg: tests-beta/android/MASVS-RESILIENCE/MASTG-TEST-0265.md

type: [static]
weakness: MASWE-0094
best-practices: []
status: new
---

## Overview

> This test checks whether the app uses `StrictMode`, which while useful for developers to log policy violations such as disk I/O or network operations in production apps, can expose sensitive implementation details in the logs that could be exploited by attackers.

Reviewer: ```suggestion
This test checks whether the app uses `StrictMode`. While useful for developers to log policy violations such as disk I/O or network operations during development, it can expose sensitive implementation details in the logs that could be exploited by attackers.
```

## Steps

1. Use @MASTG-TOOL-0110 to identify all instances of `StrictMode`

>    APIs.

Reviewer: Wouldn't it be better to reference a technique here?

Author: ```suggestion
1. Use @MASTG-TOOL-0110 to identify all instances of `StrictMode` APIs.
```
